docs/DeveloperGuide.md

```diff
@@ -516,12 +523,15 @@ Then, it will call the `runCommands()` method to process the user input and exec
 The output of the `commands` will be printed by the `textUi` object which is an attribute of `AtomicHabitManager` class
 and is initialised in the constructor. `habitList` which was initialised in the constructor is the `AtomicHabitList`
 object that stores all the user's habits.
+<!-- @@author -->
 
-### Managers
+<!-- @@author haoyangw -->
+## Managers
 
 ![Manager](diagrams/Manager.png)<br/>
 The `Manager` abstract class is the superclass for classes responsible for handling user interaction with the app.
 
+### Implementation Rationale
```

Comment:
Should we standardise as "Design Considerations" like what other sections used?

Reply:
Ok fixed! thanks